examples/sample-app/README.md

```diff
@@ -104,52 +104,49 @@ the present working directory is the same directory as this README.
         $ export CURL_CA_BUNDLE=`pwd`/openshift.local.certificates/admin/root.crt
         $ sudo chmod +r "$KUBECONFIG"
 
-4. Deploy a private docker registry within OpenShift with the certs necessary for access to master:
+4. Deploy a private Docker registry within OpenShift with the certs necessary for access to master:
 
         $ sudo chmod +r ./openshift.local.certificates/openshift-client/key.key
-        $ pushd ../..
-        $ CERT_DIR=examples/sample-app/openshift.local.certificates/openshift-client hack/install-registry.sh
-        $ popd
+        $ openshift ex registry --create --credentials="${KUBECONFIG}"
+          docker-registry # the service
+          docker-registry # the deployment config
 
-    Note that the private docker registry is using ephemeral storage,
+    Note that the private Docker registry is using ephemeral storage,
     so when it is stopped, the image will be lost. An external volume
     could be used for persistent storage, but that is beyond the scope
     of this tutorial.
 
 5. Confirm the registry is started (this can take a few minutes):
 
-        $ osc get pods
+        $ osc describe service docker-registry
```

Comment:
you can tell because it returns endpoints, which is the equivalent to listing the pods.  (also the curl command still gets run to test it in step 6)